ledger/sandbox-on-x/src/test/lib/scala/com/daml/platform/sandbox/auth/ReadWriteServiceCallAuthTests.scala

```diff
@@ -16,63 +17,107 @@ trait ReadWriteServiceCallAuthTests extends ServiceCallWithMainActorAuthTests {
     createUserByAdmin(userPrefix + mainActor, Vector(proto.Right(right)))
       .flatMap { case (_, token) => serviceCallWithoutApplicationId(token) }
 
-  it should "deny calls with an expired read/write token" in {
+  it should "deny calls with an expired read/write token" taggedAs securityAsset.setAttack(
+    attack(threat = "Exploit an expired read/write token")
+  ) in {
     expectUnauthenticated(serviceCallWithToken(canActAsMainActorExpired))
   }
-  it should "allow calls with explicitly non-expired read/write token" in {
+  it should "allow calls with explicitly non-expired read/write token" taggedAs securityAsset
+    .setHappyCase(
+      "Ledger API client can connect with explicitly non-expired read/write token"
```

Comment:
```suggestion
      "Ledger API client can make a call with explicitly non-expired read/write token"
```
Similar changes across this file